module/Olcs/src/Controller/Messages/LicenceCloseConversationController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Olcs\Controller\Messages;
+
+use     Common\Controller\Interfaces\ToggleAwareInterface;
```

Comment:
v.minor whitespace issue here.

Reply:
You say minor, I say instantly drove me crazy.